src/Compilers/Core/MSBuildTask/Shared/ManagedCompiler.cs

```diff
@@ -60,6 +60,9 @@ public ITaskItem[] Analyzers
         // We do not support BugReport because it always requires user interaction,
         // which will cause a hang.
 
+        /// <summary>
+        /// An instrument flag that specifies classes for which the compiler should generate instrumentation information.
```

Comment:
As @AlekseyTs suggested, this is probably overly specific.  For now, it would be clearer to say something like "specifies instrumentation settings".